app/graphql/types/mutation_type.rb

```diff
@@ -3,6 +3,7 @@ class MutationType < Types::BaseObject
     field :create_characteristic, mutation: Mutations::Characteristics::CreateCharacteristic
     field :create_character, mutation: Mutations::Characters::CreateCharacter
     field :create_skill, mutation: Mutations::Skills::CreateSkill
+    field :create_user, mutation: Mutations::Users::CreateUser
```

Comment:
Curious, why is this nested under `CreateSkill`?

Reply:
Moved this up just to keep all the `creates` together BUT would be happy to move it down as the most recent mutation, wasn't sure what convention was with this!

Reply:
Idk what the convention is either but it's way more readable this way!